### PR TITLE
libhttpseverywhere: download rules

### PR DESCRIPTION
**Pull request by kencu**
rather than having them as a large file
in the files dir

### CODE REVIEW

**Comment by kencu** [comment]
@jmroot -- I think this is right. Works.

File: www/libhttpseverywhere/Portfile [hunk 1]
```diff
@@ -23,14 +23,29 @@ long_description    ${description}. HTTPSEverywhere is a browser plugin \
...
+set rules_commit    9c43785fe5ce13c70b8a1abfece834d12fc9d0fb
+
+extract.only        ${main_distfile}
+
+distfiles           ${main_distfile}:main \
+                    ${rules_distfile}:rules
+
+homepage            https://git.gnome.org/browse/${name}
+master_sites        gnome:sources/${name}/${branch}/:main \
+                    'https://gitlab.gnome.org/GNOME/libhttpseverywhere/-/raw/${rules_commit}/data/default.rulesets?inline=false&dummy=':rules
```
Comment (jmroot) [inline www/libhttpseverywhere/Portfile:39]: You can lose the single quotes here; they don't have any special meaning in Tcl.

**Review by jmroot — CHANGES_REQUESTED** [review]

File: www/libhttpseverywhere/Portfile (hunk 1 above)
Comment (kencu) [reply www/libhttpseverywhere/Portfile]: done.

**Review by jmroot — APPROVED** [review]
LGTM

**Comment by kencu** [comment]
thanks

**Pull request merged**